broker/src/test/java/org/apache/rocketmq/broker/filter/MessageStoreWithFilterTest.java

```diff
@@ -238,7 +238,7 @@ public void testGetMessage_withFilterBitMapAndConsumerChanged() {
 
             // sleep to wait for consume queue has been constructed.
             try {
-                Thread.sleep(1000);
+                Thread.sleep(2001);
```

Comment:
Looks there is no such interface, maybe keep polling till it finds something for a period of time and claiming failure if timeout.